console - npm install - vendor/moment-locale-de.js compiled, but not written

As seen on https://github.com/georchestra/georchestra/runs/423184127
```
[INFO] --- frontend-maven-plugin:1.6:install-node-and-npm (install-node-and-npm) @ console ---
[INFO] Node v10.18.0 is already installed.
[INFO] NPM 6.13.4 is already installed.
[INFO] 
[INFO] --- frontend-maven-plugin:1.6:npm (npm install) @ console ---
[INFO] Running 'npm install --unsafe-perm' in /home/runner/work/georchestra/georchestra/console/src/main/webapp/manager
[INFO] 
[INFO] > manager@0.0.1 postinstall /home/runner/work/georchestra/georchestra/console/src/main/webapp/manager
[INFO] > standard app/**/*.{js,es6} && ./node_modules/.bin/brunch build
[...] 
[ERROR] 13:34:06 - warn: vendor/moment-locale-de.js compiled, but not written. Check your javascripts.joinTo config
[ERROR] 13:34:06 - warn: vendor/moment-locale-es.js compiled, but not written. Check your javascripts.joinTo config
[ERROR] 13:34:06 - warn: vendor/moment-locale-fr.js compiled, but not written. Check your javascripts.joinTo config
[INFO] 13:34:06 - info: compiling
[INFO] 13:34:06 - info: compiled 76 files into 6 files, copied 13 in 4.3 sec
[WARNING] npm WARN manager@0.0.1 No license field.
[WARNING] npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules/fsevents):
[WARNING] npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
```

--- a/console/src/main/webapp/manager/app/app.es6
+++ b/console/src/main/webapp/manager/app/app.es6
@@ -143,7 +143,10 @@ angular.module('manager', [
     }])
   .filter('dateFormat', ['$translate', ($translate) => {
     moment.locale($translate.use())
-    return date => moment(date).format('LL')
+    return (date, format) => {
+      const m = moment(date)
+      return `<span title="${m.format('lll')}">${!format ? m.fromNow() : m.format(format)}</span>`
+    }
   }])
   .directive('shortname', () => ({
     require: 'ngModel',

--- a/console/src/main/webapp/manager/app/components/logger/logger.tpl.html
+++ b/console/src/main/webapp/manager/app/components/logger/logger.tpl.html
@@ -74,7 +74,7 @@
           <td>
             <span ng-if="!log.changed" title="{{::log.title}}">
                 <i class="glyphicon glyphicon-{{::log.icon}}"></i>
-                {{::log.type}}
+                {{::'logs.'+log.type.split('_').join('').toLowerCase() | translate}}
             </span>  
             <span ng-if="log.changed && log.type=='EMAIL_SENT'">
               <i class="glyphicon glyphicon-envelope" title="{{::log.title | translate}}"></i>

--- a/console/src/main/webapp/manager/app/components/logger/logger.tpl.html
+++ b/console/src/main/webapp/manager/app/components/logger/logger.tpl.html
@@ -50,7 +50,7 @@
         </thead>
     <tbody>
         <tr dir-paginate="log in logger.logs | logs: logger.type:logger.admin:logger.target:logger.date | orderBy: '-date' | itemsPerPage: logger.itemsPerPage">
-          <td>{{::log.date | logDate}}</td>
+          <td><abbr ng-bind-html="log.date | dateFormat"></abbr></td>
           <td><a ng-link="user({id: log.admin, tab: 'infos'})" title="">{{::log.admin}}</a>
           <!--Target-->
           <td>

--- a/console/src/main/webapp/manager/app/components/user/user.tpl.html
+++ b/console/src/main/webapp/manager/app/components/user/user.tpl.html
@@ -33,7 +33,7 @@
       </div>
 
       <div class="panel-heading text-center" ng-if="user.user.expired">
-        <span translate>user.expiredmsg</span> : {{::user.user.shadowExpire | dateFormat: 'LL'}}
+        <span translate>user.expiredmsg</span> : <span ng-bind-html="user.user.shadowExpire | dateFormat: 'LL'"></span>
       </div>
 
       <div class="panel-body" ng-if="user.tab=='infos'">

--- a/console/src/main/webapp/manager/brunch-config.js
+++ b/console/src/main/webapp/manager/brunch-config.js
@@ -38,7 +38,10 @@
               'vendor/ol.js',
               'vendor/FileSaver.js',
               'vendor/promise.min.js',
-              'vendor/fetch.js'
+              'vendor/fetch.js',
+              'vendor/moment-locale-de.js',
+              'vendor/moment-locale-fr.js',
+              'vendor/moment-locale-es.js'
             ]
           }
         },

